base/array.jl

```diff
@@ -1615,8 +1615,14 @@ julia> findnext(A,3)
 function findnext(A, start::Integer)
     l = endof(A)
     i = start
+    warned = false
```

Comment:
Won't this warn everytime the function is called?

Reply:
I was counting on `depwarn` for that.

Reply:
Yeah, this looks good to me — the `warned` makes sure the slower `depwarn` is only called once for each invocation.

Reply:
Oh, sorry for the noise.